bottlecap/src/lifecycle/invocation/processor.rs

```diff
@@ -574,15 +574,31 @@ impl Processor {
             self.span.error = 1;
 
             if let Some(m) = message {
+                let decoded_message = match base64_to_string(m) {
+                    Ok(decoded) => decoded,
+                    Err(e) => {
+                        debug!("Failed to decode error message: {e}");
```

Comment:
Message is too ambiguous, since you're actually setting the error at the end, so it's kinda misleading and user might think that it failed, but the error message is actually there